### PR TITLE
Added: support to set current location from PO details, shipment details page and return page (#238pmgz)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/components/LocationPopover.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    })
+  },
+  props: ['item', 'type', 'facilityId'],
+  methods: {
+    setFacilityLocation(event: CustomEvent) {
+      const facilityLocations = this.getFacilityLocationsByFacilityId(this.facilityId)
+      if (facilityLocations) {
+        const facilityLocation = facilityLocations.find((location: any) => location.locationSeqId === event['detail'].value)
+        if(facilityLocation) {
+          this.store.dispatch(`${this.type}/setItemLocationSeqId`, { item: this.item, facilityLocation });
```
Comment (ymaheshwari1) [inline src/components/LocationPopover.vue:41]: Pass locationSeqId directly from here instead of passing facilityLocation

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/ReturnDetails.vue [hunk 2]
```diff
@@ -80,6 +91,7 @@ import {
   IonToolbar,
   modalController,
   alertController,
+  onIonViewWillEnter,
```
Comment (ymaheshwari1) [inline src/views/ReturnDetails.vue:94]: I think this is unused.

File: src/views/ReturnDetails.vue [hunk 3]
```diff
@@ -109,21 +122,30 @@ export default defineComponent({
...
     IonToolbar,
-    Image
+    Image,
+    LocationPopover
   },
   props: ["shipment"],
   data() {
     return {
-      queryString: ''
+      queryString: '',
```
Comment (ymaheshwari1) [inline src/views/ReturnDetails.vue:131]: Remove unwanted change.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/ReturnDetails.vue (hunk 3 above)
Comment (disha1202) [reply src/views/ReturnDetails.vue]: Removed

File: src/views/ReturnDetails.vue (hunk 2 above)
Comment (disha1202) [reply src/views/ReturnDetails.vue]: Removed unused code

File: src/components/LocationPopover.vue (hunk 1 above)
Comment (disha1202) [reply src/components/LocationPopover.vue]: Made the requested change

File: src/views/ReturnDetails.vue [hunk 4]
```diff
@@ -25,29 +26,39 @@
...
           </div>
-
+            
+  
           <ion-item class="border-top" v-if="item.quantityOrdered > 0">
             <ion-button @click="receiveAll(item)" slot="start" fill="outline">
               {{ $t("Receive All") }}
             </ion-button>
-            <ion-progress-bar :value="item.quantityAccepted/item.quantityOrdered" />
+            <ion-progress-bar :value="item.quantityAccepted/item.quantityOrdered"></ion-progress-bar>
```
[the remark this answers is not in the document]
Comment (disha1202) [reply src/views/ReturnDetails.vue]: Done Sir

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**